apps/vth-frontend/src/styles/globals.css

```diff
@@ -153,6 +153,35 @@ a {
   max-width: var(--utrecht-page-content-max-width);
 }
 
+@media screen and (-ms-high-contrast: active), screen and (forced-colors: active) {
+  .utrecht-accordion__header .utrecht-accordion__button {
```

Comment:
Moet de selector echt zo specifiek zijn, of zou `.utrecht-accordion__button` ook al werken? Geldt ook voor alle andere selectors hier.

Reply:
Nee, als ik de eerste class wegdoe, dan gaat het design kaputt.

Reply:
Ik denk dat de accordion style moet worden geüpdate vanuit de designstyle package, zodat ook de VTH dezelfde stijl kan krijgen.
https://github.com/nl-design-system/utrecht/blob/main/components/accordion/css/_mixin.scss

Reply:
@Robbert 
Wat denk jij?